lang/cgilua/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cgilua
+PKG_VERSION:=5.1.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/keplerproject/cgilua/archive/
+PKG_MD5SUM:=ed162ca341d0f1cd16868c80bed3da4c
+
+PKG_MAINTAINER:=Alex R <alex@totalwebservices.net>
+PKG_LICENSE=MIT
+
+LUA_MODULE_PATH:=/usr/lib/lua
+LUA_BIN_PATH:=/usr/bin
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/cgilua
+  SUBMENU:=Lua
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=cgilua
+  URL:=https://github.com/keplerproject/cgilua
+  DEPENDS:=+lua +luafilesystem
+endef
+
+define Package/cgilua/description
+	CGILua is a tool for creating dynamic HTML pages and manipulating input data from Web forms.
+endef
+
+define Build/Configure
+endef
```

Comment:
Can you please clarify?  Why?  Which block exactly?  I'm pretty sure define Package/cgilua is needed.  Maybe define Build/Configure?